plugins/colored-man-pages/colored-man-pages.plugin.sh

```diff
@@ -0,0 +1,47 @@
+#! bash oh-my-bash.module
+
+# Safety get the colors.
+if tput setaf 1 &> /dev/null; then
+  COLORED_MAN_PAGES_BLACK=$(tput setaf 0)
+  COLORED_MAN_PAGES_RED=$(tput setaf 1)
+  COLORED_MAN_PAGES_GREEN=$(tput setaf 2)
+  COLORED_MAN_PAGES_YELLOW=$(tput setaf 3)
+  COLORED_MAN_PAGES_BLUE=$(tput setaf 4)
+  COLORED_MAN_PAGES_MAGENTA=$(tput setaf 5)
+  COLORED_MAN_PAGES_CYAN=$(tput setaf 6)
+  COLORED_MAN_PAGES_WHITE=$(tput setaf 7)
+  COLORED_MAN_PAGES_BOLD=$(tput bold)
+  COLORED_MAN_PAGES_RESET=$(tput sgr0)
```

Comment:
We already have variables `_omb_term_{black,brown,green,olive,navy,purple,teal,silver,bold,reset}`. We don't want to call a bunch of `tput` for every single plugin. It slows down the initialization. Since those variables `COLORED_MAN_PAGES_*` are not intended to be used by users, we can simply remove these variables. Instead one can directly specify `$_ble_term_*` to `LESS_TERMCAP_*`.